Add documentation for RAID and BIOS feature (#35658)
openshift/openshift-docs: modules/ipi-install-configuring-bios-for-worker-node.adoc

// Module included in the following assemblies:
//
// * installing/installing_bare_metal_ipi/ipi-install-configuration-files.adoc
[id="configuring-bios-for-worker-node_{context}"]
= Configuring BIOS for worker node

> During the installation process, the BIOS of the worker node can be configured

Reviewer: ```suggestion
The following procedure configures BIOS for the worker node during the installation process. 
```

Author: updated, thanks.

by the following steps.

> 

Reviewer: Are there any prerequisites for this procedure that needs to be included? Such as needing `cluster-admin` privileges?

Author: There is no prerequisite, which is the normal cluster deployment through IPI.

.Procedure
. Create manifests.
. Modify the BMH file corresponding to the worker:
+
----
$ vim clusterconfigs/openshift/99_openshift-cluster-api_hosts-3.yaml
----
. Add the BIOS configuration to the `spec` section of the BMH file:
+
----
spec:
  firmware:
    simultaneousMultithreadingEnabled: true
    sriovEnabled: true
    virtualizationEnabled: true
----
+
[NOTE]
====

> . Only support these three BIOS configurations. See the

Reviewer: ```suggestion
Red Hat supports three BIOS configurations. See the link:https://github.com/openshift/baremetal-operator/blob/master/docs/api.md#firmware[BMH documentation] for details. Only servers with bcc type `irmc` are supported. Other types of servers are currently not supported. 
```

I would put this in 1 NOTE versus making it layered since it's the same type of content.

Author: > Other types of servers are currently not supported. 

From the code point of view, there are some other types of servers that support this feature, but we don't have a machine for testing. So I think it's better to explain that other types of services are not tested, rather than  not supported.

Author: updated, PTAL.

Reviewer 2: thanks the update.
I think RH can't support not tested servers, so this modification is needed.

// Module included in the following assemblies:
//
// * installing/installing_bare_metal_ipi/ipi-install-configuration-files.adoc
[id="configuring-bios-for-worker-node_{context}"]

> 

Reviewer 3: Remove whitespace.

Author: Thanks, updated, PTAL.

= Configuring BIOS for worker node

During the installation process, the BIOS of the worker node can be configured
by the following steps.

.Procedure

> . Create manifests

Reviewer 3: Add a period at the end. If there is an example, add a colon, and then provide an example of creating the manifests.

> . Modify the BMH file corresponding to the worker

Reviewer 3: Add a colon at the end.

+
----
$ vim clusterconfigs/openshift/99_openshift-cluster-api_hosts-3.yaml
----

> . Add BIOS configuration in `spec`:

Reviewer 3: Add the BIOS configuration to the `spec` section of the BMH file:

+
----
spec:
  firmware:
    simultaneousMultithreadingEnabled: true
    sriovEnabled: true
    virtualizationEnabled: true
----
+
[NOTE]
====
. Only support these three BIOS configurations. See the

> link:https://github.com/metal3-io/baremetal-operator/blob/master/docs/api.md#firmware[BMH documentation] for details.

Reviewer 3: We are not allowed to link to upstream repos as sources. So the link needs to be removed and details brought downstream.
